webgui/src/flipmouse/js/elecutils.js

```diff
@@ -0,0 +1,102 @@
+window.electronUtils = {};
+
+//guard this whole file for running in electron
+if (C.IS_ELECTRON) {
```

Comment:
`if (!C.IS_ELECTRON) { return; }` nicer since it doesn't need the whole file to be encapsulated in the if-block.

Reply:
Good point -> changed.
Still some C thinking :-)

Reply:
I tried this, but I got "Illegal return statement" -> keep it this way...

Reply:
OK, yes, since you are not in a function, you cannot return ;)